### DISCUSSION TITLE
Remove stochastic scenenario from output data

### DISCUSSION THREAD

**Discussion by DillonJ** [spine-tools/SpineOpt.jl#862] [Ideas]
Stochastic data is input as map type, why not output stochastic data as map types also.

The vast majority of models are deterministic and seeing stochastic scenario as a dimension to all output data is confusing for users and creates a lot of unnecessary clutter and inconveniently long class names.

I think there are two potential solutions to this:

 - Output stochastic data as map type (consistent with input data) and determinsitic outputs as timeseries (with no stochastic scenario dimension).
 - Detect if model has only one unqiue stochastic scenario and if so, remove stochastic scenario as an output dimension.

#430 relates

**Comment by mihlema** [comment]
If I'm not mistaken, we already support output as Maps for runs which save the analysis time? (E.g. rolling window optimization that saves the entire window for each roll)
For me, it seems okay to go about the stochastic in a similar way, but I'm also not too sure how nice this will be for looking at results if e.g. we have both stochastics and analysis time maps? (Map in Map)

**Comment by clizbe** [comment]
@DillonJ Has this been addressed or have we moved on?
